### DISCUSSION TITLE
Promethus Unable to view new targets

### DISCUSSION THREAD

**Discussion by chunkushisan** [prometheus/prometheus#14602] [Q&A]
### What did you do?

I downloaded Prometheus from the Operator, kube-prometheus-release 0.13.zip, then, I created a static service and a monitor to monitor windows outside the K8S cluster, but I can't see anything in targets

Here are some of my config files

job_name: 'blackbox'
params:
  module: [http_2xx]
static_configs:
- targets:
  - http://gaoxin.kubeasy.com
  - https://www.baidu.com
relabel_configs:
- source_labels: [__address__]
  target_label: __param_target
- source_labels: [__param_target]
  target_label: instance
- target_label: __address__
  replacement: blackbox-exporter:19115
metrics_path: /probe
# -job_name: "node-exporter-external"

This is configured statically

-----------------------------------------------------------------------------
apiVersion: v1
kind: Service
metadata:
  name: windows-exporter-157
  namespace: prometheus-exporter
  labels:
    app: windows-exporter-157
    app.kubernetes.io/name: windows-exporter-157
spec:
  clusterIP: None
  ports:
  - name: metrics
    port: 9182
    protocol: TCP
    targetPort: 9182

----------------------------------------------------
apiVersion: v1
kind: Endpoints
metadata:
    name: windows-exporter-157
    namespace: prometheus-exporter
    labels:
      app: windows-exporter-157
      app.kubernetes.io/name: windows-exporter-157
subsets:
- addresses:
  # 这里是外部资源列表
  - ip: 10.1.121.157
  ports:
  - name: metrics
    port: 9182
--------------------------------------------
apiVersion: monitoring.coreos.com/v1
kind: ServiceMonitor
metadata:
  name: windows-exporter-157
  namespace: prometheus-exporter
  labels:
    app: windows-exporter-157
    release: prometheus
spec:
  selector:
    matchLabels:
      app: windows-exporter-157
  namespaceSelector:
    matchNames:
      - prometheus-exporter
  endpoints:
    - port: metrics
      interval: 1m
      path: /metrics
      scheme: http
--------------------------------------------

### What did you expect to see?

Because I have just come into contact with Prometheus, I don't know a lot. I hope you can answer my questions accurately to the command steps. Thank you

### What did you see instead? Under which circumstances?

targest has no monitoring items

**Comment by machine424** [comment]
I suggest you use slack for such "support" questions https://prometheus.io/community/
